.changeset/nine-paws-unite.md

```diff
@@ -0,0 +1,12 @@
+---
+'sku': major
+---
+
+`sku init` no longer installs and configures husky for you
+
+BREAKING CHANGE
+
+`sku init` no longer adds `husky` as a dependecy nor does it configure husky for you out of the box.
```

Comment:
```suggestion
`sku init` no longer adds `husky` as a dependecy nor does it configure Husky for you out of the box.
```

![old man](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fnewfastuff.com%2Fwp-content%2Fuploads%2F2019%2F06%2Fa7PqaZO.png&f=1&nofb=1&ipt=b792475f8390ac32e75542f405fb9cde27056303fd4a7b107a7345fa977cde24&ipo=images)

Reply:
Actually, in this one I'm the old man 😢